docs/src/modules/ROOT/pages/integration/config-solver-properties.adoc

```diff
@@ -0,0 +1,52 @@
+////
+Quarkus and Spring Boot support the same configuration properties.
+All the properties are in this file, which can then be included multiple times.
+The {property_prefix} attribute is used for Quarkus properties.
+////
+
+{property_prefix}timefold.solver.<solverName>.solver-config-xml::
+A classpath resource to read the solver configuration XML.
+Defaults to `solverConfig.xml`.
+If this property isn't specified, that file is optional.
+
+{property_prefix}timefold.solver.<solverName>.environment-mode::
+Enable runtime assertions to detect common bugs in your implementation during development.
+
+{property_prefix}timefold.solver.<solverName>.daemon::
+Enable xref:responding-to-change/responding-to-change.adoc#daemon[daemon mode].
+In daemon mode, non-early termination pauses the solver instead of stopping it, until the next problem fact change arrives.
+This is often useful for xref:responding-to-change/responding-to-change.adoc#realTimePlanning[real-time planning].
+Defaults to `false`.
+
+{property_prefix}timefold.solver.<solverName>.move-thread-count::
+Enable multi-threaded solving for a single problem, which increases CPU consumption.
+Defaults to `NONE`.
+Note that this is a feature of the xref:enterprise-edition/enterprise-edition.adoc[Enterprise edition],
+which is Timefold's commercial offering.
+See xref:enterprise-edition/enterprise-edition.adoc#multithreadedIncrementalSolving[multithreaded incremental solving].
+
+{property_prefix}timefold.solver.<solverName>.domain-access-type::
+How Timefold Solver should access the domain model.
+See xref:using-timefold-solver/configuration.adoc#domainAccess[the domain access section] for more details.
+ifeval::["{property_prefix}" == "quarkus."]
+Defaults to `GIZMO`.
+The other possible value is `REFLECTION`.
+endif::[]
+ifeval::["{property_prefix}" == ""]
+Defaults to `REFLECTION`.
+The other possible value is `GIZMO`.
```

Comment:
Since this is for Spring Boot, I recommend adding a note on Gizmo limitations outside of Quarkus.
If we have this in the docs already, a link will be enough.
Otherwise we really should explain how GIZMO - for some minor performance benefits - requires you to open your domain model, making fields public.